plugins-compat-tester/src/main/java/org/jenkins/tools/test/hook/PipelineStageViewHook.java

```diff
@@ -2,28 +2,34 @@
 
 import hudson.model.UpdateSite;
 import java.util.Map;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
 import org.jenkins.tools.test.model.PomData;
 
-public class PipelineRestApiHook extends AbstractMultiParentHook {
+public class PipelineStageViewHook
+    extends AbstractMultiParentHook {
+
+    private static final Logger LOGGER = Logger.getLogger( PipelineStageViewHook.class.getName());
 
     @Override
     protected String getParentFolder() {
         return "pipeline-stage-view";
     }
 
     @Override
-    protected String getParentUrl() {
-        return "scm:git:git://github.com/jenkinsci/pipeline-stage-view-plugin.git";
+    protected String getPluginFolderName( UpdateSite.Plugin currentPlugin ) {
+        return "ui";
```

Comment:
I think this needs to handle both plugins, since neither of them uses the default format where the artifactId is the plugin folder name.

```suggestion
        return currentPlugin.name.equals("pipeline-rest-api") ? "rest-api" : "ui";
```